tests/scripts/pulp_rpm/test_rpm_sync_publish.sh

```diff
@@ -27,12 +27,13 @@ expect_succ pulp rpm repository show --name "cli_test_rpm_repository"
 test "$(echo "$OUTPUT" | jq -r '.description')" = "null"
 
 # skip-types is broken in 3.12.0
-if pulp debug has-plugin --name "rpm" --min-version "3.12.0" --max-version "3.12.1"
-then
-  expect_succ pulp rpm repository sync --name "cli_test_rpm_repository"
-else
-  expect_succ pulp rpm repository sync --name "cli_test_rpm_repository" --skip-type srpm
-fi
+#if pulp debug has-plugin --name "rpm" --min-version "3.12.0" --max-version "3.12.1"
+#then
+#  expect_succ pulp rpm repository sync --name "cli_test_rpm_repository"
+#else
+#  expect_succ pulp rpm repository sync --name "cli_test_rpm_repository" --skip-type srpm
+#fi
+expect_succ pulp rpm repository sync --name "cli_test_rpm_repository"
```

Comment:
Why this change? (The conditional was probably for a bug with srpms in 3.12.0 fixed in 3.12.1 .)
If you think we do not need to work around anymore, please remove the commented part.

Reply:
`Error: ["Cannot use 'retain_package_versions' in combination with a 'mirror_complete' or 'mirror_content_only' sync policy."]`